tests/component-tests/textfieldControl/js/TextfieldStepperTests.js

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid, jqUnit */
+
+(function ($) {
+    "use strict";
+
+    $(document).ready(function () {
+
+        /**********************************
+         * Textfield Stepper Button Tests *
+         **********************************/
+
+        jqUnit.module("Textfield Stepper Button Tests");
+
+        fluid.defaults("fluid.tests.textfieldStepper.button", {
+            gradeNames: ["fluid.textfieldStepper.button"],
+            strings: {
+                label: "Button Label"
+            }
+        });
+
+        jqUnit.test("Test Init", function () {
+            jqUnit.expect(2);
+            var that = fluid.tests.textfieldStepper.button(".flc-textfieldStepper-button");
+
+            jqUnit.assertEquals("The label should be set", that.options.strings.label, that.container.attr("aria-label"));
```

Comment:
Please also verify the button's `aria-label` and tab order have been set correctly.